docs/guide-disappearance.mdx

```diff
@@ -32,7 +32,7 @@ test('movie title appears', async () => {
 
   // wait for appearance inside an assertion
   await waitFor(() => {
-    expect(getByText('the lion king')).toBeInTheDocument()
+    expect(screen.getByText('the lion king')).toBeInTheDocument()
```

Comment:
We encourage the usage of `findBy` over this, so should this example be added to the guide?
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find

Reply:
Hey @fildon, do you want to update the guide by removing this section (`2. Using waitFor`) please?

Reply:
Can do, but then I don't think this section of the docs flows well any more. 

This example is number 2 in a pair of examples. The first example demonstrates "findByText" and this one demonstrates "waitFor". If we want to change this one to use "findByText" then we are just duplicating the two examples effectively. 

Perhaps the second example should be removed entirely? And then this part of the docs shouldn't be a numbered list (since then it would have only one entry)?

Reply:
Good catch, I think that's a good proposition 👍

Reply:
Maybe this example is too simple, but there are still reasons to use waitFor in a positive case, such as batching together several element queries in one retry block or adding in additional assertions after the getBy that also need to rerun (ie, checking element properties are in the right state).

Reply:
So you would leave it like this, or should it have a comment/more complex example?